terraform/aks/workspace_variables/review_aks.tfvars.json

```diff
@@ -1,11 +1,14 @@
 {
   "paas_app_environment": "review",
+  "azure_resource_prefix": "s189t01-att-rv",
   "key_vault_resource_group": "s189t01-att-rv-rg",
   "key_vault_name": "s189t01-att-rv-kv",
   "key_vault_app_secret_name": "APPLY-APP-SECRETS-REVIEW",
   "key_vault_infra_secret_name": "BAT-INFRA-SECRETS-QA",
   "cluster": "test",
+  "postgres_subnet": "postgres-review-apps-snet",
```

Comment:
Can the variable be removed? The subnet name is constant now (postgres-snet)

Reply:
removed